Purchases/Purchasing/StoreProduct.swift

```diff
@@ -106,10 +108,28 @@ public typealias SK2Product = StoreKit.Product
 
     @objc public override var localizedTitle: String { underlyingSK2Product.displayName }
 
+    @objc public override var priceFormatter: NumberFormatter? {
```

Comment:
Maybe document that this creates a new formatter every time, which can be slow?

Reply:
good call. I'll add a `TODO:` as well as a reminder to tackle it before shipping ideally

Reply:
https://github.com/RevenueCat/purchases-ios/issues/1033